src/modules/loadData.js

```diff
@@ -0,0 +1,25 @@
+import { Movies } from './resource.js';
+
+async function displayData() {
```

Comment:
- [ ] Since you are using ES6, It would be better to use arrow functions instead of the word `function`.